src/resources/postcss/rsvp/_attendee-registration.pcss

```diff
@@ -206,5 +214,10 @@
 
 	.tribe-tickets__rsvp-ar-form-title {
 		color: var(--color-text-secondary);
+
+
+		&[data-guest-number]::after {
+			content: " " attr(data-guest-number);
```

Comment:
this will only work for those languages that adds the guest number after the word "guest". If a language has the number in the middle of words (Chinese for example, 第2位客人) or at the beginning this will not work.

Instead, it would be better to see if we can get a translated string from the BE with a placeholder, pass it to the FE as well as the window object, and use it in the template and JS.

Reply:
I though of that at first, but the number is dynamic. So the translated string should be a "template" and I didn't find a way to do that, because using `%s` or whatever would be interpreted by gettext. How do you think we could use a translatable template?

Reply:
We can definitely dynamically generate the number via JS, but I think we should add a placeholder in the translated string and target that in the JS, maybe something like:

```
__( 'Guest ##guest_number##', 'event-tickets' );
```

In the JS, we can target for the placeholder `##guest_number##` and replace that with the data value. I don't know the best placeholder to use, but maybe something to chat with G or S.

Reply:
Yep, I thought of something similar, but honestly I never seen placeholders like that on translation catalogs. I think that could be a good approach.

@sc0ttkclark and @bordoni: wanna chime in so we can define all together which way to go here?

Reply:
I think `##guest_number##` would be fine there

Reply:
Actually, I think we could established placeholders for PHP (%s, %d) and then use sprintf() to convert it to a JS-friendly placeholder. That way the strings could be shared elsewhere from the PHP side and JS would still get what it needs.

Reply:
I would like to use Underscore template here so we that we have escaped and more power if we need anything.

PHP would look like: 
```
sprintf( _x( 'Guest %1$s', 'explain', domain ), '<%- guest_number %>' )
```

Than on the JS  side:
```
_.template( 'Guest <%- guest_number %>', { guest_number: '1' } )
```

I confirmed with Juan on a call that is a good way to go if everyone is in agreement.

Reply:
That seems to do it, just submitted the changes!